src/darker/tests/test_main.py

```diff
@@ -261,14 +267,22 @@ def test_main(
     expect_stdout,
     expect_a_py,
     expect_retval,
+    root_as_cwd,
+    tmp_path_factory,
 ):  # pylint: disable=too-many-arguments
     """Main function outputs diffs and modifies files correctly"""
-    monkeypatch.chdir(git_repo.root)
+    if root_as_cwd:
+        cwd = git_repo.root
+        pwd = ""
+    else:
+        cwd = tmp_path_factory.mktemp("not_a_git_repo")
+        pwd = str(git_repo.root) + "/"
+    monkeypatch.chdir(cwd)
     paths = git_repo.add({"a.py": newline, "b.py": newline}, commit="Initial commit")
     paths["a.py"].write_bytes(newline.join(A_PY).encode("ascii"))
     paths["b.py"].write_bytes("print(42 ){newline}".encode("ascii"))
 
-    retval = darker.__main__.main(arguments + ['a.py'])
+    retval = darker.__main__.main(arguments + [pwd + "a.py"])
```

Comment:
Here, a `pathlib.Path` concatenation can be used:
```suggestion
    retval = darker.__main__.main(arguments + [str(pwd  / "a.py")])
```

I've avoided many Windows compatibility gotchas this way, although in this particular case the forward slash didn't break anything in the Windows build. But maybe it's good to do this for consistency.